mne/viz/tests/test_ica.py

```diff
@@ -220,19 +221,27 @@ def test_plot_ica_sources(raw_orig, browser_backend):
     ica = ICA(n_components=2)
     ica.fit(raw, picks=ica_picks)
     ica.exclude = [1]
+    if sys.platform == 'darwin':  # unknown transformation bug
+        monkeypatch.setenv('MNE_BROWSE_RAW_SIZE', '20,20')
```

Comment:
This was necessary to get PySide6 macOS to pass for mne-qt-browser in https://github.com/mne-tools/mne-qt-browser/pull/104, but not locally. I don't get it, but this should help stabilize things a bit hopefully 🤷